src/components/scripture/dto/scripture-reference.dto.ts

```diff
@@ -20,12 +20,14 @@ export abstract class ScriptureReferenceInput {
 
   @Field(() => Int, {
     description: 'The chapter number',
+    defaultValue: null,
```

Comment:
Does this make the field nullable in the GQL schema?

Reply:
Yes.